3_javascript/javascript1/week4/script_homework_week4.js

```diff
@@ -0,0 +1,102 @@
+
+//Voice assistant - Create a function called getReply(command). The function should return a response that corresponds to the command!
+
+let getNameString = [];
+let getNameStringJustName = [];
+let toDoList = ['reading materials', 'doing exercises'];
+let getTimer = [];
+function getReply(Command) {
```

Comment:
Attributes such as ﻿`Command` should always start with a lowercase letter. Except for React components, the capital Letters are reserved only for `types` (shadowing of Typescript 👀) and `React` components following common conventions :)